prestashoperpconnect_customize_example/__init__.py

```diff
@@ -22,8 +22,6 @@
 #
 ###############################################################################
 
-from . import backend
-from . import connector
-from . import partner
-from . import prestashop_model
+
+from . import models
```

Comment:
yes but there was a mistake. the init file that was referencing the models was already into models. And the one referencing the .py files was at the root.
That's why I change both file so that they referenced in a correct way the files